### ISSUE TITLE
Build the Model using tf.keras

### ISSUE THREAD

**Issue by HarshCasper** [HakinCodes/Malaria-Detection#103]
## Type

Advancement 

## Description

Build a Model using tf.keras. Follow these instructions to submit your Notebook: 

- Use the restructured dataset which already has train/val/test set. 
- Create a  model using tf.keras 
- Evaluate your Model using Confusion Matrix 
- Test your model against the Holdout/Test Set 
- Use proper Jupyter Markdown to document your Code

## Tools

- Jupyter Notebook
- Jupyter Markdown
- Python
- Tensorflow
- Keras 

### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/HakinCodes/Malaria-Detection/blob/master/CONTRIBUTING.md)?

Yes

**Comment by kritikaparmar-programmer** [comment]
I want to work on this issue.. @HarshCasper

**Comment by HarshCasper** [comment]
Assigned to you @kritikaparmar-programmer

**Comment by macabdul9** [comment]
@kritikaparmar-programmer I have updated the description a little, let me know if you have any problem in modeling.
